integrationtests/src/test/java/io/kroxylicious/proxy/MetricsIT.java

```diff
@@ -32,11 +31,6 @@
 @ExtendWith(KafkaClusterExtension.class)
 public class MetricsIT {
 
-    @BeforeEach
-    public void beforeEach() {
-        Metrics.globalRegistry.clear();
-    }
-
```

Comment:
In principal I quite like the fact that the test is ensuring the slate is clean before execution even if it is strictly not required.

Reply:
How about asserting that it is before/after empty?  That would at least alert us to meters being left around.

Reply:
Hmm, that's sort of giving the tests multiple reasons to fail which I don't like, but as you say would detect leaks.